percona/pmm/pmm.go

```diff
@@ -229,10 +233,10 @@ func sidecarContainerV2(pgc *v2.PerconaPGCluster) corev1.Container {
 		},
 	}
 
-	if pgc.CompareVersion("2.3.0") >= 0 {
+	if pgc.CompareVersion("2.7.0") >= 0 {
 		container.Env = append(container.Env, corev1.EnvVar{
-			Name:  "PMM_AGENT_PATHS_TEMPDIR",
-			Value: "/tmp",
+			Name:  "PMM_POSTGRES_PARAMS",
+			Value: pmmSpec.PostgresParams,
```

Comment:
This new env var should be available most probably for PMM3 as well. We need to add it to the PMM3 container construction.

Reply:
I see) I will update it after merging https://github.com/percona/percona-postgresql-operator/pull/1100